osquery/core/init.cpp

```diff
@@ -699,6 +699,10 @@ void Initializer::start() const {
                      FLAGS_numeric_monitoring_plugins);
   }
 
+  if (Killswitch::get().isAppStartMonitorEnabled()) {
+    monitoring::record("osquery.start", 1, monitoring::PreAggregationType::Sum);
```

Comment:
Would you like to consider using something like `application.start`? Just to be clear. Because every monitoring key there is about osquery, why only this certain key consist the name of project. Does it make sense?

Reply:
I don't think this is important. With the same logic everything should start with application.